packages/rpc/src/rpc-clusters.ts

```diff
@@ -5,7 +5,6 @@ import type { ClusterUrl, DevnetUrl, MainnetUrl, TestnetUrl } from '@solana/rpc-
 export type RpcTransportDevnet = RpcTransport & { '~cluster': 'devnet' };
 export type RpcTransportTestnet = RpcTransport & { '~cluster': 'testnet' };
 export type RpcTransportMainnet = RpcTransport & { '~cluster': 'mainnet' };
-export type RpcTransportWithCluster = RpcTransportDevnet | RpcTransportMainnet | RpcTransportTestnet;
```

Comment:
I couldn't find or think of a use for this. In particular, the `Rpc*` types below have no such thing.